wp-includes/rest-api/endpoints/class-wp-rest-customize-changesets-controller.php

```diff
@@ -610,6 +747,13 @@ protected function prepare_item_for_database( $request ) {
 				}
 
 				if ( isset( $data[ $setting_id ] ) ) {
+
+					// If the value of the setting is null, this should be removed from the changeset. @todo confirm logic.
+					if ( null === $data[ $setting_id ] ) {
```

Comment:
@westonruter Any thoughts on how should it be possible to remove existing values from changeset when updating? Maybe if the `params` is either set to `"null"` or just the `value` is set to `null` or if `params` is empty, maybe then? Thoughts?